src/extension/Pty.ts

```diff
@@ -34,8 +39,13 @@ export default class Pty {
     this.pid = this.pty.pid;
 
     // Set prompt
-    this.pty.write(`export PS1='${UUID}|$?|$(pwd)|'\r`);
-    this.pty.write(`export PS2='${PS2}'\r`);
+    if (shell.endsWith("csh")) {
```

Comment:
This information should probably come from the profile.

If you pass in `graph.profile` to [new Pty](https://github.com/AndreasArvidsson/bashbook/blob/f3047c3859a39b6dddba4f515095ac27898234be/src/extension/Notebook.ts#L42) like `new Pty(shell, cwd, graph.profile)`

Then in [Profile](https://github.com/AndreasArvidsson/bashbook/blob/f3047c3859a39b6dddba4f515095ac27898234be/src/extension/profiles/Profile.ts#L5) you can add the functions getPS1(): string` and `getPS2(): string`.